admin/templates/Includes/LeftAndMain_MenuStatus.ss

```diff
@@ -3,7 +3,7 @@
 	<% with $CurrentMember %>
 		<span>
 			<%t LeftAndMain_Menu_ss.Hello 'Hi' %>
-			<a href="{$AbsoluteBaseURL}{$AdminURL}admin/myprofile" class="profile-link">
+			<a href="{$AdminURL}myprofile" class="profile-link">
```

Comment:
Hmm, my bad, I thought there were docs examples showing to do it this way, but on inspection I don't see them